lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -58,6 +59,7 @@
                 <!-- Actions (left) -->
                 <div *ngIf="actions && actionsPosition === 'left'" class="adf-datatable-table-cell">
                     <button mat-icon-button [matMenuTriggerFor]="menu"
+                            role="cell"
```

Comment:
It's not a cell, it's a button within the cell. Maybe needs to have "div role=cell" instead?

Reply:
I guess Denys is right it here there are some info about it 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Grid_Role
This gridcell should be in the upper div